src/EmitJs.re

```diff
@@ -185,6 +185,10 @@ let emitCodeItems = (~language, ~outputFileRelative, ~resolver, codeItems) => {
         );
       let moduleNameBs = moduleName |> ModuleName.forBsFile;
 
+      let fnParam = (paramName, paramType) => switch(language) {
```

Comment:
The case analysis over language is normally performed inside module EmitTyp.
For this specific use case, see the other uses of EmitTyp.ofType.